cmd/github-mcp-server/main.go

```diff
@@ -69,6 +68,7 @@ func init() {
 	rootCmd.PersistentFlags().StringSlice("toolsets", github.DefaultTools, "An optional comma separated list of groups of tools to allow, defaults to enabling all")
 	rootCmd.PersistentFlags().Bool("dynamic-toolsets", false, "Enable dynamic toolsets")
 	rootCmd.PersistentFlags().Bool("read-only", false, "Restrict the server to read-only operations")
+	rootCmd.PersistentFlags().Bool("read_only", false, "Restrict the server to read-only operations")
```

Comment:
I don't think we need the flag here, we can just allow viper env parser to fix the env version.